src/main/java/com/hubspot/jinjava/JinjavaConfig.java

```diff
@@ -125,6 +128,10 @@ public boolean isFailOnUnknownTokens() {
     return failOnUnknownTokens;
   }
 
+  public boolean isEnableNestedInterpretation() {
```

Comment:
good point. That was mine. :) Maybe rename in a separate PR so as not to clutter up this one.